File: forloop/code.php
<?php

$i_counter = 0;

for ($i_loop1 = 0; $i_loop1 < 10; $i_loop1++) {
   for ($i_loop2 = 0; $i_loop2 < 32000; $i_loop2++) {
       for ($i_loop3 = 0; $i_loop3 < 32000; $i_loop3++) {
           $i_counter++;
           if ($i_counter > 50) {
               $i_counter = 0;
           }
       }
   } 
}

echo 'Counter: '.$i_counter."\n";

File: forloop/code.rs
fn main() {

  let mut i_counter: i16 = 0;
  for i_loop1 in 1..=31999 {
    for i_loop2 in 1..=31999 {
      for i_loop3 in 1..=31999 {
        i_counter += 1;
				if i_counter > 50 {
					i_counter = 0;
				}
      }
    }
  }

  println!("{}", i_counter);

}